src/dotnet/Chat.UI.Blazor/Components/ChatListListenedRemove.razor

```diff
@@ -0,0 +1,28 @@
+@namespace ActualChat.Chat.UI.Blazor.Components
+@inherits ComputedStateComponent<ImmutableArray<Symbol>>
+@{
+    var cls = "pin-toggle";
+}
+
+<ButtonRound Class="@cls" Click="@OnClick" Title="Remove from Listened">
+    <svg class="fill-current w-6 h-6" xmlns="http://www.w3.org/2000/svg" viewBox="2 1 22 22">
```

Comment:
Not for this iteration, but IMO we should start creating our own icon lib - with a common base class for every icon w/ standard size settings.

Reply:
It's a bit annying when all these huge blocks are everywhere around.